reliability/level3/README.md

```diff
@@ -0,0 +1,10 @@
+# Level 3
+
+## Summary
+
+The level3 program is the same as level2.
```

Comment:
That doesn't feel right, there are some differences

Reply:
There isn't the files are literally identical, but I think I understand where you're coming from, I'll add something in the base README.me about the level helpers.